resource/localization/ru/hl_camera_tool.properties

```diff
@@ -4,6 +4,7 @@ hl_camera.roll=\u0421\u0432\u0435\u0440\u043d\u0443\u0442\u044c
 hl_camera.nearz=\u0411\u043b\u0438\u0437\u043a\u0438\u0439\u0020\u005a
 hl_camera.farz=\u0414\u0430\u043b\u044c\u043d\u0438\u0439\u0020\u005a
 hl_camera.viewoffset=\u0421\u0434\u0432\u0438\u0433\u0020\u0412\u0438\u0434\u0430
+hl_camera.enableghost=\u041F\u043E\u043A\u0430\u0437\u0020\u0441\u0434\u0432\u0438\u0433\u0430\u0020\u0432\u0438\u0434\u0430
```

Comment:
I trust you that the russian is correct :)

Reply:
Rather than "Enable ghost" this says "Show view offset", just in case